modules/components/pages/outputs/cypher.adoc

```diff
@@ -0,0 +1,409 @@
+= cypher
+:type: output
+:status: experimental
+:categories: ["Services"]
+
+// © 2024 Redpanda Data Inc.
+
+
+component_type_dropdown::[]
+
+
+Writes a batch of messages to any graph database that supports the Neo4j or Bolt protocols.
+
+Introduced in version 4.37.0.
```

Comment:
We should conditionalize this in case we ever want to single-source it into cloud docs.

Reply:
Updated as requested.